src/main/scala/beam/utils/google_routes_db/doc/README.md

```diff
@@ -0,0 +1,29 @@
+# TravelTimeGoogleStatistic: Build database of the computed routes
```

Comment:
Should you document how to update the db config - or add something to gradle to generate it when the base beam config is updated?